README.md

```diff
@@ -53,15 +53,18 @@ Fork this repository to your GitHub account.
    + Integer, mandatory, no default.
 
 
+
 2. Commit and push the changes to your forked repository.
 
-### Configure Application Properties
-1. Change the value of `jira-issue-lottery.config-file-repo` in the `application.properties` file to point to a public repository with a valid config file according to first step in [Configuration section](#configure-jira-lottery) such as `https://github.com/jboss-set/jira-issue-lottery`.
 
 ### Create .env File
 1. Create a `.env` file in the main directory of your app.
-2. Create a Jira Personal Access Token ([Guide on how to create the token](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html))
-3. Add the following variable to the `.env` file:
+2. Add the value of `%dev.jira-issue-lottery.config-file-repo` in the `.env` file to point to a public repository with a valid config file according to first step in [Configuration section](#configure-jira-lottery) such as `https://github.com/jboss-set/jira-issue-lottery`.
```

Comment:
Why do you need to move it out of application.properties? If it's environment variable than the naming should be adjusted to make it clear - e.g., THIS_IS_ENV_VAR